jekyll/_cci2/testing-llm-enabled-applications-through-evaluations.adoc

```diff
@@ -64,42 +61,72 @@ The link:https://circleci.com/developer/orbs/orb/circleci/evals[official Evals o
 
-Link to tutorial which will live in another page ( under Tests > Tutorials)
-Need code example in CircleCI-public
+You can configure test case assertions such as:
 
-You can find a full tutorial of how to use the Evals orb on this page.
-////
+* Thresholds: Check whether 1 or more results fall within a range.
++
+[,shell]
+----
+{
+  “TestCaseName”: “correctness > 0.99 && toxicity < 0.01”
+}
+----
+
+* Equality: Check whether results provide an expected answer.
++
+[,shell]
+----
+{
+  “TestCaseName”: “label == \”CORRECT\””
+}
+----
+
+Evaluation testing results determine if a job should stop or continue:
+
+* **If the evaluation test fails**: Indicates a proposed change resulted in a degradation of model performance. The job will stop running, and the pipeline will fail.
+* **If the evaluation test passes**: Indicates that model performance has met set criteria, the job will continue running.
 
-Further documentation on how to use the orb is available on the link:https://circleci.com/developer/orbs/orb/circleci/evals[Orb page]. Orb usage examples are available in the sample repository link:https://github.com/CircleCI-Public/llm-eval-examples[[LLM-eval-examples].
+==== View evaluation test results
 
-== Storing credentials for your evaluations
-CircleCI makes it easy to store your credentials for LLM providers as well as LLMOps tools. Navigate to menu:Project Settings[LLMOps] to enter, verify, and access your OpenAI secrets. You will also find a starting template for your `config.yml` file.
+Evaluation testing determines if your evaluation results meet specified conditions. Evaluation testing results are presented in the CircleCI web UI in two locations:
 
-You can also save the credentials to your preferred evaluation platform, including Braintrust and LangSmith. These credentials can then be used when setting up a pipeline that leverages the Evals orb.
+* In the step details
++
+image::/docs/assets/img/docs/llmops/eval-job-eval-test-step.png[Evaluation testing: Job Step Details]
+
+* In the tests tab. Additionally, when a test case has failed, its details are displayed.
++
+image::/docs/assets/img/docs/llmops/eval-test-fail-detail.png[Evaluation testing: Test Failure Details]
 
-To get started, navigate to menu:Project Settings[LLMOps]:
+== Store credentials for your evaluations
+CircleCI makes it easy to store your credentials for LLM providers as well as LLMOps tools.
 
-image::/docs/assets/img/docs/llmops/create-context.png[Jobs overview]
+* Navigate to **Project Settings > LLMOps** to enter, verify, and access your OpenAI secrets.
+* Here, you can also find a starting template for your `config.yml` file.
+* You can save the credentials for your evaluation platform, including Braintrust and LangSmith. These credentials can then be used when setting up a pipeline that uses the Evals orb.
```

Comment:
Is this something that's coming soon? I see OpenAI and Amazon Bedrock currently?

Reply:
I was also confused by this; the LLMOps page is awaiting an update.. Though adding credentials for evaluation providers is actually a subtask when adding OpenAI credentials. I altered this copy a bit based on that detail. Thoughts?

Reply:
```
To store your LLM provider credentials, follow these steps:

. Navigate to menu:Project Settings[LLMOps]
. Select btn:[Set Up] next to your chosen provider, and follow the in-app instructions.
. When connecting an OpenAI account, you can also save the credentials for your evaluation platform, such as Braintrust and LangSmith. These credentials can then be used when setting up a pipeline that uses the Evals orb.
```